values.yaml

```diff
@@ -137,6 +137,15 @@ global:
       limits: {}
       requests: {}
 
+  # External ES logging
+  custom_logging:
```

Comment:
@danielhoherd : we are yet to decide on custom_logging or external_logging to keep same name across all places.

Reply:
IMHO `custom_logging` makes more sense since it may not be external. IE: we could use this for logging to other locally hosted services.

Reply:
https://helm.sh/docs/chart_best_practices/values/#naming-conventions

Reply:
so need to move to customlogging

Reply:
@danielhoherd @vishwas-astro - pls confirm on the name

Reply:
seems like `customLogging` is it

`git grep -l custom_logging | gsed -i 's/custom_logging/customLogging/g'`

Reply:
Renamed to customLogging